src/main/java/io/strimzi/kafka/bridge/amqp/AmqpConfig.java

```diff
@@ -20,13 +20,13 @@
     private static final String AMQP_MESSAGE_CONVERTER = "AMQP_MESSAGE_CONVERTER";
```

Comment:
Do we still need these? Are these the env var definitions?

Reply:
We can use it as "amqp.message.converter" instead of the hard-coded one and making it public.